pkg/startup/startup.go

```diff
@@ -63,6 +63,7 @@ const (
 	AUTODETECTION_METHOD_INTERFACE      = "interface="
 	AUTODETECTION_METHOD_SKIP_INTERFACE = "skip-interface="
 	AUTODETECTION_METHOD_CIDR           = "cidr="
+	AUTODETECTION_METHOD_K8S            = "k8s"
```

Comment:
For future-proofing, should it be `k8s=internal-ip` or something like that?

Reply:
In the next step, we are also going to get the external IP. We pretty much require that knowledge in EKS. But we canmake it configurable this way. `k8s=internal` and `k8s=all` in the next step. WDYT?

Reply:
Agree we should start with `k8s=internal` and then add in additional methods.

If we start with just `k8s` then we'll need to maintain support for that field indefinitely. We should try to agree all of the options that we intend to write up-front.